assets/cubyz/blocks/sulfur_torch.zig.zon

```diff
@@ -1,5 +1,5 @@
 .{
-	.tags = .{.wood},
+	.tags = .{.choppable, .wood},
```

Comment:
I wouldn't give torches the wood tag.
They are crafted from wood, but that is irrelevant to their usage.